rules/windows/execution_apt_solarwinds_backdoor_child_cmd_powershell.toml

```diff
@@ -0,0 +1,52 @@
+[metadata]
+creation_date = "2020/12/14"
+maturity = "production"
+updated_date = "2020/12/14"
+
+[rule]
+author = ["Elastic"]
+description = "A suspicious SolarWinds child process (Cmd.exe or Powershell.exe) was detected."
+false_positives = [
+    "Trusted SolarWinds Child Processes, Verify process details such as network connections and file writes.",
+]
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Command Execution via SolarWinds Process"
+references = [
+    "https://www.fireeye.com/blog/threat-research/2020/12/evasive-attacker-leverages-solarwinds-supply-chain-compromises-with-sunburst-backdoor.html",
+    "https://github.com/fireeye/sunburst_countermeasures/blob/main/rules/SUNBURST/hxioc/SOLARWINDS%20SUSPICIOUS%20FILEWRITES%20(METHODOLOGY).ioc",
+]
+risk_score = 47
+rule_id = "d72e33fc-6e91-42ff-ac8b-e573268c5a87"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Execution"]
+type = "eql"
+
+query = '''
+process where event.type in ("start", "process_started") and process.name: ("cmd.exe", "powershell.exe") and
```

Comment:
Everytime this rule triggers, so will `Suspicious SolarWinds Child Process`. Is there a need to have this one focused on just these two processes versus having them taken care of in the other rule? Just trying to remove duplicate alerts.

Reply:
nope won't trigger other rule, because SolarWinds.BusinessLayerHost*.exe is not in the list here (removed for the same reason), all the process.parent.name executables in this rule are potential target to load the same backdoored DLL and to not make it to much reduce it to potential suspicious child processes (cmd.exe or powershell.exe ... both reported in Veloxity details).